docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,106 @@
+Configuration and usage
+=======================
+
+Getting around the administrator interface
+------------------------------------------
+
+The ``edx-enterprise`` project is integrated with edx-platform Django admin. It can be accessed at
+``$LMS_SERVER_ADDRESS/admin/enterprise``. Current administrator interface is considered provisional until an independent
+full-featured enterprise portal is built, so not much time was spent on improving admin tools UX.
+
+.. image:: images/enterprise-admin.png
+
+So far, there are four items in the admin interface:
+
+* "Enrollment notification email templates" - manages templates used to build enrollment notifications.
+* "Enterprise Customers" - lists and manages Enterprise Customer records.
```

Comment:
@grantgoodmanedX Rest of the docs use title case for names of entities, so I'll change it on the other line and keep this one.